src/identity/selectors/index.ts

```diff
@@ -78,11 +84,6 @@ export const selectQuartzIdentityStatus = (state: AppState): RemoteDataState =>
 export const selectQuartzBillingStatus = (state: AppState): RemoteDataState =>
   state.identity.currentIdentity.loadingStatus.billingStatus
 
-export const selectQuartzOrgDetailsStatus = (
-  state: AppState
-): RemoteDataState =>
-  state.identity.currentIdentity.loadingStatus.orgDetailsStatus
-
 export const selectQuartzOrgs = (
```

Comment:
We don't need this redux status selector anymore - we are using local state to determine when an API call should be made, because we want it to be run every time the org settings tab is loaded, even if the user navigates back later in the same session.